app/views/handlers/feedback.py

```diff
@@ -84,13 +83,14 @@ def get_page_title(self) -> str:
         return self.PAGE_TITLE
 
     def handle_post(self) -> None:
-        session_data = self._session_store.session_data
+        session_data: SessionData = self._session_store.session_data  # type: ignore
         session_data.feedback_count += 1
 
-        feedback_metadata = FeedbackMetadata(
-            session_data.case_id,
-            session_data.tx_id,
-        )
+        if session_data.case_id and session_data.tx_id:
+            feedback_metadata = FeedbackMetadata(
+                session_data.case_id,
+                session_data.tx_id,
+            )
```

Comment:
Feedback metadata needs to exist, if this statement was false, this would error as `feedback_metadata` wouldn't exist. I think it is okay to ignore it for now, we need to make a design choice first, so no harm ignoring here now given we have ignores in few places here already. We also need to review the types for metadata as case_id and tx_id should be mandatory.